src/actions.sol

```diff
@@ -57,6 +53,10 @@ contract Actions is DSNote {
         erc20.approve(usr, amount);
     }
 
+    function transferFromERC20(ERC20Like erc20, address src, address dst, uint amount) public {
+        erc20.transferFrom(src, dst, amount);
```

Comment:
So far, we always used this pattern and only passing the address in the function.
No strong feelings here but maybe we can agree on a standard.
```suggestion
        ERC20Like(erc20).transferFrom(src, dst, amount);
```

Reply:
I think we should maybe just use `transfer` instead of `transferFrom` because currently we would run into the same problem again with the open zeppelin implementation. I just realized.